crates/pop-cli/src/commands/mod.rs

```diff
@@ -45,6 +46,9 @@ pub(crate) enum Command {
 	/// Remove generated/cached artifacts.
 	#[clap(alias = "C")]
 	Clean(clean::CleanArgs),
+	/// Add a new feature to your existing polkadot-sdk project
```

Comment:
```suggestion
	/// Add a new feature to your existing parachain project
```

We call it `parachain project` in other places if my memory is correct.

Reply:
Mmm, the command is useful in any project using the polkadot-sdk, doesn't need to be created with `pop`, that's why I used that word there. You can also add a config type to a standalone pallet, so dunno if the word parachain would be more correct here